KsApi/models/Project.swift

```diff
@@ -65,7 +65,7 @@ public struct Project {
   public struct Stats {
     public var backersCount: Int
     public var commentsCount: Int?
-    public var convertedPledgedAmount: Int?
+    public var convertedPledgedAmount: Double?
```

Comment:
I think we can use `Float` here since we will still need to `round` the value. Float has lesser precision and can be "faster" to read.

Reply:
Updated.